### ISSUE TITLE
Request: Hotkey or button to "create WP at current position"

### ISSUE THREAD

**Issue by AndKe** [mavlink/qgroundcontrol#5793]
When in planning mode, Vehicle is shown on map.
We often fly over complex areas to collect outer boundaries of late-to-be surveyed area.  
(looking nadir to determine the affected area before planning survey)

It would be very neat, if we had a key/button to create a waypoint at UAV's current lat/lon/alt.  
(in QGC, not UAV)

**Comment by fingadar** [comment]
Good idea. We often have the problem that "normal" users have problems to set the polygon points for survey grid using the map.
Creating the polygon points for a survey grid with the actual position of the tablet would be perfect for them. So they could simply walk around the borders of their agricultural field, set the points simply by pushing a button on the pad and that would be perfect.
DJI showns this method in one of their promotion videos for their Argas copter.

**Comment by DonLakeFlyer** [comment]
There isn't a hotkey but you can select Edit Location from the hambuger which allows you to move an item to the vehicle location.

**Comment by Williangalvani** [comment]
This seems to the same as #4275

**Comment by AndKe** [comment]
Yes, those are basically the same.
Should be labeled as for Release 3.9 ,that would be more believable, right along with proper flight monitoring features requested years ago :)

**Comment by DonLakeFlyer** [comment]
> proper flight monitoring features

Don't remember what that is. Is there an issue somewhere?
